sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> so that the frame is immediately sent if the channel is clear

Doesn't this value back off the time for sensing the channel? If the channel is sensed clear, the packet should always be sent immediately, right?

But otherwise, after looking at the datasheet again, it makes sense, what you've written. I confused CSMA_RETRIES with FRAME_RETRIES.